test/functional/test_framework/test_framework.py

```diff
@@ -129,6 +130,8 @@ def main(self):
                             help="use bitcoin-cli instead of RPC for all commands")
         parser.add_argument("--perf", dest="perf", default=False, action="store_true",
                             help="profile running nodes with perf for the duration of the test")
+        parser.add_argument("--randseed", action="store", type=int,
```

Comment:
```suggestion
        parser.add_argument("--randseed", type=int,
```

syle-nit: No need to mention the default when it isn't done anywhere else in our code.

Reply:
removed